source/fundamentals/connection.txt

```diff
@@ -0,0 +1,25 @@
+.. _csharp-connection:
+
+==========
+Connection
+==========
+
+.. default-domain:: mongodb
+
+.. toctree::
+
+   /fundamentals/connection/connect
+   /fundamentals/connection/connection-options
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this section, you'll learn how to use the {+driver-short+} to connect your application to a MongoDB deployment. Click a link in the following list to jump to a topic: 
+
+- :ref:`How to Connect to MongoDB <csharp-connect-to-mongodb>`
```

Comment:
Will this have more links? A list of one item seems bare.

Reply:
Yes, it will link to the other pages once those have been added in separate tickets. I left them out for now so as not to introduce broken links.